T3/Gui/Interaction/SingleValueEdit.cs

```diff
@@ -390,6 +401,7 @@ private enum InputStates
 
         private static string _numberFormat = "{0:0.000}";
         private static double _timeOpened;
+        private const double DefaultValue = 0.0;
```

Comment:
should be `_defaultValue` because of private

Reply:
noted!